lib/makeWebpackConfig.js

```diff
@@ -102,6 +105,17 @@ module.exports = async (playroomConfig, options) => {
         }
       ]
     },
+    optimization: {
+      nodeEnv: options.production ? 'production' : 'development',
+      minimize: options.production,
+      concatenateModules: options.production,
```

Comment:
The top 3 options are actually the defaults so can probably drop unless you want to be explicit.